src/containers/Admin/ThemeTab/index.tsx

```diff
@@ -32,6 +59,14 @@ const ThemeTab = (): JSX.Element => {
         })
     }
 
+    const switchDirection = (value: Direction): void => {
+        setRotationRadioValue(value)
+        setSettings({
+            direction: value,
+        })
+        onChangeDirection(value)
+    }
```

Comment:
saves direction both in settings here, and local storage in onChangeDirection, redundant